.github/workflows/nix.yml

```diff
@@ -38,11 +38,14 @@ jobs:
           - "3.10"
           - "3.11"
           - "3.12"
+          - "3.13"
         include:
           - os: ubuntu-arm64-24.04 # aarch64-linux
             python-version: "3.12"
           - os: macos-14 # aarch64-darwin
             python-version: "3.12"
+          - os: macos-14 # aarch64-darwin
+            python-version: "3.13"
```

Comment:
Does this need a corresponding entry in the skip helper?

Reply:
Done.